src/oscar/core/customisation.py

```diff
@@ -45,10 +45,21 @@ def inherit_app_config(local_app_folder_path, local_app_name, app_config):
         join(local_app_folder_path, 'apps.py'),
         "import {app_config_class_module} as apps\n\n\n"
         "class {app_config_class_name}(apps.{app_config_class_name}):\n"
-        "    name = '{app_name}'\n".format(
+        "    name = '{app_name}'\n"
+        "    label = '{label}'\n".format(
             app_config_class_module=app_config.__module__,
             app_config_class_name=app_config.__class__.__name__,
-            app_name=local_app_name))
+            app_name=local_app_name,
+            label=label,
+            ))
+
+    create_file(
```

Comment:
@jayvdb is this not duplicating the same logic as inlines 39-43 above?

Reply:
Ahh , this was added since v2.0 which is where I needed it.  Very good.

What about the addition of `label`?

If not, pls close.

If it is beneficial, I'll tidy up the PR.

Reply:
The label is inherited from the parent class - I don't think things would work if you change that label at project level.

Reply:
Ya true. ok, 628bd1e10 makes this redundant then.